public/controllers/management/components/management/groups/groups-main.js

```diff
@@ -39,8 +40,20 @@ class WzGroups extends Component {
     store.dispatch(updateGlobalBreadcrumb(breadcrumb));
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     this.setGlobalBreadcrumb();
+    console.log(window.location.href)
+    // Check if there is a group in the URL
+    const [_, group] = window.location.href.match(new RegExp('group=' + '([^&]*)')) || [];
+    window.location.href = window.location.href.replace(new RegExp('group=' + '[^&]*'), '');
+    if(group){
+      try{
+        // Try if the group can be accesed
+        const responseGroup = await WzRequest.apiReq('GET', '/groups', {params: {groups_list: group}});
+        const dataGroup = responseGroup?.data?.data?.affected_items?.[0];
```

Comment:
`?.` it's ok? we don't have problems whits these?

Reply:
Yes, it should work now when the app package is built.